test/cets_test_peer.erl

```diff
@@ -1,16 +1,61 @@
 -module(cets_test_peer).
 -export([
-    start_node/1,
+    start/2,
+    stop/1,
     node_to_peer/1
 ]).
+
+-export([
+    block_node/2,
+    reconnect_node/2,
+    disconnect_node/2,
+    disconnect_node_by_name/2
+]).
+
+-import(cets_test_rpc, [rpc/4]).
+
 -include_lib("common_test/include/ct.hrl").
 
-start_node(Sname) ->
+start(Names, Config) ->
```

Comment:
Minor, but this argument is called an `Id` in other places, like the `find_or_start_node/1` function. From what I see, it's used more like and ID, so I'd go with this name for this argument everywhere.